ballerina/modules/http_status/http_status_errors.bal

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
```

Comment:
Why this is changed?

Reply:
AFAIR in latest license headers we add this as `.com` right?